Doc/whatsnew/3.13.rst

```diff
@@ -742,6 +742,11 @@ email
   (Contributed by Thomas Dwyer and Victor Stinner for :gh:`102988` to improve
   the :cve:`2023-27043` fix.)
 
+enum
+----
+
+* :class:`~enum.EnumDict` has been made public in :mod:`enum`.
```

Comment:
_What's New_ should provide a quick overview to the casual reader. It might be useful to note that `EnumDict` was previously private, but I would expect that most people who know that already know what it is used for -- can we rephrase here to say that a new public type has been added that can be used in XYZ scenarios? You can also add a credit note for Ethan and a link to the GH issue (https://github.com/python/cpython/issues/112328).

```suggestion
* :class:`~enum.EnumDict` has been made public in :mod:`enum`.
```